src/test/java/marquez/common/models/NamespaceNameTest.java

```diff
@@ -33,6 +33,12 @@ public void testNewNamespace() {
     assertEquals(value, NamespaceName.fromString(value).getValue());
   }
 
+  @Test
```

Comment:
I don't anticipate issues, but please also add a resource-level test.

Reply:
I just modified the namespace name use in the NameSpaceResourceTest to make sure.

Reply:
That works too! Thanks :)